### DISCUSSION TITLE
revalidateIfStale is not clear in doucmentation? What does it mean? What does it mean if data is stale?

### DISCUSSION THREAD

**Discussion by roker15** [vercel/swr#1747] [General]
revalidateIfStale is not clear in doucmentation? What does it mean? What does it mean by data is stale? If there is cache already there , is it stale or not? another thing what if revalidateOnMount contradicts revalidateIfStale. Is is possible to fetch data when tab is opened first time and then it stops fetching until i close the tab and reopens?

**Comment by MonstraG** [comment]
See https://github.com/vercel/swr/discussions/1400
